From 23df88cef9aee2101e852f32812450df93ef77eb Mon Sep 17 00:00:00 2001
From: Arun Ravindran <arun.ravindran@intel.com>
Date: Wed, 18 Sep 2013 15:05:03 +0300
Subject: [PORT FROM MAIN] uevent: set telephony permission/ownsership

BZ: 94560

Change-Id: I8f71bd7fd95942d3efa4f460380ebd5a0eeeefaf
Orig-Change-Id: I7a99d119c0041d178f1802c0ea86e60fe39ea17f
Category: device enablement
Domain: Telephony-FWK
Origin: internal
Upstream-Candidate: no
Signed-off-by: Arun Ravindran <arun.ravindran@intel.com>
---
 rootdir/ueventd.rc |    9 +++++++++
 1 files changed, 9 insertions(+), 0 deletions(-)

diff --git a/rootdir/ueventd.rc b/rootdir/ueventd.rc
index eff24c3..903de39 100644
--- a/rootdir/ueventd.rc
+++ b/rootdir/ueventd.rc
@@ -82,6 +82,15 @@ subsystem adf
 /dev/mtp_usb              0660   root       mtp
 /dev/usb_accessory        0660   root       usb
 /dev/tun                  0660   system     vpn
+/dev/ttyACM*              0660   root       radio
+/dev/ttyPTI1              0660   root       radio
+/dev/gsmtty*              0660   root       radio
+/dev/ttyMFD*              0660   root       radio
+/dev/ttyGS0               0660   root       radio
+/dev/mdmTrace             0660   root       radio
+/dev/mdm_ctrl             0660   root       radio
+/dev/ttyIFX0              0660   root       radio
+/dev/ttyIFX1              0660   root       radio
 
 # CDMA radio interface MUX
 /dev/ts0710mux*           0640   radio      radio
-- 
1.7.4.1

